test/unit/specs/components/common/Page404.spec.js

```diff
@@ -8,13 +9,19 @@ describe("Page404", () => {
   beforeEach(() => {
     let test = instance.mount(Page404)
     wrapper = test.wrapper
+    wrapper.update()
   })
 
-  it("has the expected html structure", () => {
+  it("has the expected html structure", async () => {
+    await wrapper.vm.$nextTick()
+    wrapper.update()
+    // console.log(wrapper.vm.$el)
```

Comment:
?

Reply:
$nextTick is added for the scrollbar plugin